_src/app/Identify.js

```diff
@@ -44,60 +52,63 @@ define([
 
             lang.mixin(this, params);
 
-            var width = 300;
-            var height = 400;
-
-            this.map.on('click', lang.hitch(this, 'onMapClick'));
-            this.map.infoWindow.setTitle('Map Click Information');
-            this.map.infoWindow.resize(width, height);
+            this.mapView.on('click', lang.hitch(this, 'onMapClick'));
```

Comment:
we shouldn't have any more lang.hitch calls. `bind` should be the path forward since there is no ie8 support in 4x.